### PR TITLE
March 6 2024 - Stefan Spieker spotlight

### PR DESCRIPTION
**Pull request by kmartens27**
This pull request is for the contributor spotlight page for Stefan Spieker (@StefanSpieker).

It does not have the `page-features:false` update for the previous spotlight, as it has not been published yet. However, once the previous spotlight has been published, this PR will be updated accordingly.

@StefanSpieker would you be able to review when you have time, and confirm whether or not that image is what you would like used?

### CODE REVIEW

File: src/pages/contributors/stefan-spieker.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+:page-github: StefanSpieker
+:page-threads:
+:page-email: 
+:page-image: avatar/stefan-spieker.png
+:page-pronouns: He/Him/His
+:page-location: Herzogenrath, Germany
+:page-firstcommit: 2011
+:page-datepublished: 2024-03-06
+:page-featured: true
+:page-intro: Stefan Spieker started early with software development, selling his first software while in 9th grade. As a result result of this early success, his software was used for more than 10 years. At that point, it was a clear path for Stefan to study computer science at the RWTH Aachen University in Aachen, Germany. His first jobs provided experiences that led him to picking up Jenkins, which he then tried to reciprocate by introducing his teams to the project.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:14]:
```suggestion
:page-intro: Stefan Spieker started early with software development, selling his first software while in 9th grade. As a result result of this early success, his software was used for more than 10 years. At that point, it was a clear path for Stefan to study computer science at the RWTH Aachen University in Aachen, Germany. His first jobs provided experiences that led him to picking up Jenkins, that he tried to reciprocate by introducing his teams to the project.
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Though he switched roles to be a solution architect, he continued to contribute to Jenkins in his free time. When he's not advocating for Jenkins, Stefan enjoys being active by playing volleyball and badminton weekly. Stefan has also written a book for people who use computers constantly, but never got a computer science education titled “52 Stunden Informatik” (52 Hours of Computer Science). In the book, there is a dedicated chapter for open source, further displaying Stefan's affinity for the open-source community and its projects.
+
+== What is your background prior to contributing to Jenkins?
+
+In my first job we did a lot without a build server, just writing make files and executing them manually.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:20]:
```suggestion
In my first job we did a lot without a build server, we just wrote makefiles and executed them manually.
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+In my next job, we had a build server which was pretty complicated - link:https://cruisecontrolnet.org/[cruisecontrolnet.org].
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:21]:
```suggestion
In my next job, we had a build server that was pretty complicated - link:https://cruisecontrolnet.org/[cruisecontrolnet.org].
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+I really love the openness of the project.
+I was also able to motivate and help colleagues contribute to Jenkins and other open-source projects, which feels really great.
+I think open source is a very important part of the software business and Jenkins is a great opportunity to get a really good impression of how it should work.
+I believe that almost everyone can and should participate in open-source projects.
+
+== What problems has Jenkins solved for you?
+
+All my automation challenges were solved with Jenkins.
+From traditional software building and testing to automating database maintenance or notifications about new tool releases.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:47]:
```suggestion
For example, from traditional software building and testing to automating database maintenance or notifications about new tool releases.
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+The flexibility of Jenkins with this huge number of plugins, over 2500 members, and even more contributors.
+I joined a Contributor summit at FOSDEM in 2020 in person, which was really great to actually meet others working on Jenkins.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:52]:
```suggestion
I joined a Contributor summit at FOSDEM in 2020 in person and that was really great to actually meet others who work on Jenkins.
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+This motivated me a lot to do even more contributions.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:53]:
```suggestion
This motivated me a lot to do even make more contributions.
```

File: src/pages/contributors/stefan-spieker.adoc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+Meanwhile, I changed my role to a solution architect, where I did not program as much as I would like to.
+This felt a little bit like Batman: taking a lot of meetings during the day, but doing the right thing at night by providing some contributions to the open-source community, mainly Jenkins.
+This also helps me keep my feet close to the ground, because I still feel strongly connected to the Jenkins community and their experience when it comes to software development.
+
+== What sort of contributions have felt the most successful or impactful?
+
+I've done a lot of quality related PRs and finally got Jenkins core to zero open SpotBugs issues, with great help from other contributors and maintainers of course.
```
Comment (ammanders) [inline src/pages/contributors/stefan-spieker.adoc:60]:
```suggestion
I've created a lot of quality-related PRs and finally got Jenkins core to zero open SpotBugs issues, with great help from other contributors and maintainers, of course.
```

**Review by ammanders — APPROVED** [review]

**Review by ammanders — APPROVED** [review]

**Review by StefanSpieker — APPROVED** [review]
Thanks again @kmartens27 for this great article and putting everything together. I really like the contributor spotlight and reading about the other contributors.
And of course I'm ok with the picture

**Review by krisstern — APPROVED** [review]
LGTM

**Review by alyssat — APPROVED** [review]
Another interesting contributor story :o) . I don't see anything that would require edit/change imo.

**Pull request merged**